packages/shared/src/components/onboarding/OnboardingPlusStep.tsx

```diff
@@ -111,17 +48,23 @@ const PlusSkeleton = (): ReactElement => (
 export const OnboardingPlusStep = ({
   onClickNext,
 }: OnboardingStepProps): ReactElement => {
-  const { productOptions, earlyAdopterPlanId } = usePaymentContext();
-  const [currentProductIndex, setCurrentProductIndex] = useState<number>(0);
   const isLaptop = useViewSize(ViewSize.Laptop);
-
-  const items = useMemo(
-    () => productOptions.filter(({ value }) => value !== earlyAdopterPlanId),
-    [productOptions, earlyAdopterPlanId],
+  const { productOptions } = usePaymentContext();
+  const item = useMemo(
+    () =>
+      productOptions
+        .toSorted(({ appsId, duration }) => {
+          if (appsId === PlusPriceTypeAppsId.EarlyAdopter) {
```

Comment:
Kindly ask the product team to update the "Custom Data" for Early Adopter to have the ID we defined here. I think they might not have that yet.